resource_container/ResourceContainer.py

```diff
@@ -334,7 +334,12 @@ def identifier(self):
         if 'identifier' in self.resource and self.resource['identifier']:
             return self.resource['identifier']
         if 'id' in self.resource and self.resource['id']:
-            return self.resource['id']
+            id = self.resource['id']
```

Comment:
@richmahn - OK, this is going back to my original approach.  I just misunderstood your suggestion.